packages/edit-site/src/components/block-editor/back-button.js

```diff
@@ -22,11 +23,12 @@ function BackButton() {
 	const history = useHistory();
 	const isTemplatePart = location.params.postType === TEMPLATE_PART_POST_TYPE;
 	const isNavigationMenu = location.params.postType === NAVIGATION_POST_TYPE;
+	const isPattern = location.params.postType === PATTERN_TYPES.user;
 	const previousTemplateId = location.state?.fromTemplateId;
 
-	const isFocusMode = isTemplatePart || isNavigationMenu;
```

Comment:
It would be nice at some point to replace this with the `hasHistory` type check rather than multiplying post type checks, but  that can be a future follow-up.